MIGRATION_GUIDE.md

```diff
@@ -9,6 +9,14 @@ across different versions.
 
 ## v0.99.0 ➞ v0.100.0
 
+### unsafe_execute resource changes / new execute resource
+
+The `snowflake_unsafe_execute` gets deprecated in favor of the new resource `snowflake_execute`.
+It will be removed before v1, so please migrate to the `snowflake_execute` resource. For no downtime migration,
```

Comment:
nit: removed before v1 -> removed with the V1 release

Reply:
Done